src/keys.hh

```diff
@@ -109,6 +117,18 @@ constexpr Key ctrlalt(Key key)
 {
     return { key.modifiers | Key::Modifiers::ControlAlt, key.key };
 }
+constexpr Key ctrlshift(Key key)
```

Comment:
same thing as for the modifiers, just let the user write `ctrl(alt(key))`, and remove `ctrlalt`